src/runtime/policy/policy.ts

```diff
@@ -197,6 +197,17 @@ export class PolicyTarget {
       return Capabilities.create(ranges);
     });
   }
+
+  getRestrictedFields() {
```

Comment:
One line comment would be good. I think it's something like:
// Returns policy fields in the format of Schema fields.
But I may be missing some subtlety.

If I'm correct however, I think having a word restricted here is adding unnecessary burden on the reader. The fields of the policy by definition are restricting, so when I read `getRestrictedFields()` I presume you are further restricting the fields in the policy, which I don't think you are doing.

Looking at the rest of this class:
`PolicyTarget.toCapabilities()`
`PolicyTarget.toMaxAge()`
`PolicyTarget.toRetention(`)

I wonder if `PolicyTarget.toSchemaFields()` would be a better name, as it would convey that you are exposing the fields of the policy type in a different format, as opposed to performing some transformation where some fields will not be described at all.

WDYT?

Reply:
+1 I agree

Reply:
Done.